upup/pkg/fi/cloudup/openstack/floatingip.go

```diff
@@ -137,3 +137,33 @@ func (c *openstackCloud) ListL3FloatingIPs(opts l3floatingip.ListOpts) (fips []l
 	}
 	return fips, nil
 }
+
+func (c *openstackCloud) DeleteFloatingIP(id string) (err error) {
+
+	done, err := vfs.RetryWithBackoff(writeBackoff, func() (bool, error) {
```

Comment:
Nit: If this is only called from the `kops delete cluster` code, you might do better not to retry here, and allow the deletion framework to retry for you.  https://github.com/kubernetes/kops/blob/master/pkg/resources/ops/delete.go#L134

That logic isn't perfect, but it does mean that you're not _required_ to retry in every delete operation.